content/components/accordions.mdx

```diff
@@ -1,22 +1,37 @@
 ---
 title: "Accordions"
-description: "A dropdown component to toggle content"
+description: "A dropdown component to toggle content visibility"
 icon: "square-caret-down"
 ---
 
 <Accordion title="I am an Accordion.">
-  You can put any content in here. Check out
-  [AccordionGroup](/content/components/accordion-groups) if you want to group
-  multiple Accordions into a single display.
+  You can put any content in here, including other components, like code:
+   ```java HelloWorld.java
+    class HelloWorld {
+        public static void main(String[] args) {
+            System.out.println("Hello, World!");
+        }
+    }
+    ```
 </Accordion>
 
+Check out [AccordionGroup](/content/components/accordion-groups) if you want to group multiple Accordions into a single display.
```

Comment:
Can we move this to be below the props by chance?

Reply:
done in [fb1487f](https://github.com/mintlify/docs/pull/633/commits/fb1487f0ecf88f79f247a5976d371b997c659d87)